packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -214,19 +223,36 @@ export async function scheduleTarget(
     'angular.json',
     workspaces.createWorkspaceHost(fsHost)
   );
+  const architectHost = await getWrappedWorkspaceNodeModulesArchitectHost(
+    workspace,
+    root,
+    opts.projects
+  );
+
+  const builderName = workspace.projects
+    .get(opts.project)
+    ?.targets?.get(opts.target)?.builder;
+  if (!builderName) {
+    throw new Error(
+      `Cannot find target '${opts.target}' for project '${opts.project}'`
+    );
+  }
+
+  const isAngularBuild =
+    builderName.startsWith('@angular/build:') ||
```

Comment:
Check if this is also necessary for `@angular-devkit/*

Reply:
No. It's only used for `@angular/build` builders:

https://github.com/angular/angular-cli/blob/f9de11d67d3e0e0524372819583bc77756596d4f/packages/angular/cli/src/command-builder/architect-base-command-module.ts#L54-L57
https://github.com/angular/angular-cli/blob/f9de11d67d3e0e0524372819583bc77756596d4f/packages/angular/cli/src/command-builder/architect-base-command-module.ts#L164-L168